admin/resources/js/components/Table.tsx

```diff
@@ -82,7 +82,6 @@ function Table<T extends Record<string, unknown>>({
           <div data-h2-flex-grid="b(middle, expanded, flush, m)">
             <div data-h2-flex-item="b(1of1) m(1of3)">
               <GlobalFilter
-                preGlobalFilteredRows={preGlobalFilteredRows}
```

Comment:
With this removal, [line 60 of this file](https://github.com/GCTC-NTGC/gc-digital-talent/pull/431/files/bb72ce5387b7d65bcb64665a0a71ac2485d09b31..eba6eba27731a7e46c6e64187c943d0c8d6bf565#diff-e1e060ddd846c8f9ab91f75034694e1bfe31685ba6471c1379b71b5841f8c757L60) now has: `preGlobalFilteredRows` is declared but its value is never read. Should be removed there as well.